# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../specter/specter-1.4-0004-PATH_MAX.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Added limits.h include to specter.h, as we use PATH_MAX
Origin: Upstream (http://bazaar.launchpad.net/~ruby/specter/trunk/diff/12)

=== modified file 'include/specter/specter.h'
--- specter-1.4/include/specter/specter.h	2008-02-25 01:34:02 +0000
+++ specter-1.4/include/specter/specter.h	2008-06-17 09:15:14 +0000
@@ -12,6 +12,7 @@
 #ifndef _SPECTER_H
 #define _SPECTER_H
 
+#include <limits.h>
 #include <sys/socket.h>
 #include <net/if.h>
 #include <linux/netfilter_ipv4/ipt_ULOG.h>

